hdbt.theme

```diff
@@ -1476,3 +1482,22 @@ function hdbt_preprocess_paragraph__school_search(array &$variables): void {
     $variables['#attached']['drupalSettings']['helfi_school_search']['cookie_privacy_url'] = $privacyUrl->toString();
   }
 }
+
+/**
+ * Implements hook_preprocess_eu_cookie_compliance_popup_info().
+ */
+function hdbt_preprocess_eu_cookie_compliance_popup_info(array &$variables) {
+  $primaryLanguages = [
+    'fi',
```

Comment:
Why is this defined second time in same file?

Reply:
Probably from doing similar things same time.
I believe the `$variables['alternative_language']` from the `hdbt_preprocess()` should be available here to be used in the if statement.